src/spec/validitygenerator.py

```diff
@@ -438,12 +438,17 @@ def createValidationLineForParameterIntroChunk(self, blockname, param, params, t
         if asciidoc != '':
             asciidoc += '\n'
 
-            # Add additional line for non-optional bitmasks
-            if self.getTypeCategory(paramtype.text) == 'bitmask':
-                if param.attrib.get('optional') is None:
+        # Add additional line for non-optional bitmasks
+        if self.getTypeCategory(paramtype.text) == 'bitmask':
+            if param.attrib.get('optional') is None:
+                isConstArrayOrPointer = (self.paramIsArray(param) or self.paramIsPointer(param)) and param.text is not None and 'const' in param.text;
+                isValue = not self.paramIsArray(param) and not self.paramIsPointer(param)
+                if isValue or isConstArrayOrPointer:
                     asciidoc += self.makeAnchor(blockname, paramname.text, 'requiredbitmask')
                     if self.paramIsArray(param):
                         asciidoc += 'Each element of '
+                    elif self.paramIsPointer(param):
+                        asciidoc += 'The value pointed by '
```

Comment:
Shoot I missed this since it's not actually showing up in the spec yet. This should be changed to "The value pointed at by" before it gets merged. Or actually just remove it - there's no obvious reason we'd ever have a const pointer to a bitmask in the spec, it makes zero sense.

Reply:
Actually should probably be marked `optional` in the xml instead of doing this guesswork here in the script based on const.

Reply:
Updated with that in mind. Those two are now `optional="false,true"`.